html/partials/500.shtml

```diff
@@ -1,2 +1,10 @@
-<h2>500 Internal server error</h2>
-<p>There's a problem. Please try again in a few moments.</p>
+<p><strong><h1></h1>Something's gone wrong with our servers</h1></strong></p>
```

Comment:
This strikes me as bad HTML relative to what it is replacing.

Reply:
`<h1></h1>Something's...</h1>`